apps/src/applab/dropletConfig.js

```diff
@@ -1131,6 +1132,16 @@ export var blocks = [
   }
 ];
 
+if (experiments.isEnabled(experiments.APPLAB_ML)) {
+  blocks.push({
+    func: 'getPrediction',
+    parent: api,
+    category: 'Data',
+    paletteParams: ['model', 'testValues', 'callback'],
+    params: ['"myModel"', 'testValues', 'function (value) {\n \n}']
+  });
+}
```

Comment:
Still TBD where this block should live, likely in it's own toolbox; but, there's open discussion about how close to keep objects to ML (e.g. should object blocks live with the other variables or are they scoped to only an ML toolbox) and now that we're using a generic `getPrediction` block will it be strange to have a toolbox with only one block in it, so punting for now, given that the block is behind an experiment flag anyway.